test/parallel/test-child-process-execfile.js

```diff
@@ -99,3 +99,20 @@ const execOpts = { encoding: 'utf8', shell: true };
   });
   execFile(process.execPath, [fixture, 0], { signal }, callback);
 }
+
+// Verify the execFile() stdout is the same as execFileSync().
+{
+  const file = 'echo';
+  const args = ['foo', 'bar'];
+
+  // Test with and without `{ shell: true }`
+  [null, { shell: true }].forEach((options) => {
+    const execFileSyncResult = execFileSync(file, args, options);
```

Comment:
Nit: would `execFileSyncStdout` will make more sense here? When I read `execFileSyncResult`, I always think: if it is `stdout` or contains a property named `stdout`(`{ stdout: "foo bar" }`). I am not familiar with the `execFile`.